app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -66,8 +64,7 @@ protected void onCreate(final Bundle savedInstanceState) {
              */
             @Override
             public void onClick(final View view) {
-                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
-                        .setAction("Action", null).show();
+                startBluetoothPairing();
```

Comment:
I think the code is more readable if you put the code in startBluetoothPairing in onClick. If you have a very long method you should split it up in multiple helper methods, but if the code is very short it doesn't really make much sense I think